client/plugins/fcc-create-nav-data/create-navigation-node.test.js

```diff
@@ -9,10 +9,10 @@ describe('fcc-create-nav-data', () => {
       internal: {
         content:
           '---\ntitle: File Writing\n---\n## File Writing\n\nThis is a stub.' +
-          " <a href='https://github.com/freecodecamp/guides/tree/master/src/" +
+          " <a href='https://github.com/freeCodeCamp/guides/tree/master/src/" +
           "pages/php/functions/files/writing/index.md' target='_blank' " +
           "rel='nofollow'>Help our community expand it</a>.\n\n<a href=" +
-          "'https://github.com/freecodecamp/guides/blob/master/README.md' " +
+          "'https://github.com/freeCodeCamp/guides/blob/master/README.md' " +
```

Comment:
```suggestion
          "'https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/style-guide-for-guide-articles.md' " +
```
Style guide is now found here for guide articles